app/src/main/res/values-sv/strings.xml

```diff
@@ -0,0 +1,167 @@
+    <string name="menu_item_title_delete_favorite">Ta bort</string>
+    <string name="menu_item_title_set_alarm">Sätt alarm</string>
+    <string name="menu_item_title_delete_alarm">Ta bort alarm</string>
+    <string name="menu_item_title_delete_all">Ta bort alla</string>
+    <string name="menu_item_title_favorites">Favoriter</string>
+    <string name="menu_item_title_feedback">Feedback</string>
+    <string name="menu_item_title_navigate">Navigering</string>
+    <string name="menu_item_title_refresh">Uppdatera</string>
+    <string name="menu_item_title_share_session_text">som text</string>
+    <string name="menu_item_title_share_session_json">som JSON (för Chaosflix)</string>
+    <string name="share_as_text">Dela som text</string>
+    <string name="share_as_json">Dela som JSON (för Chaosflix)</string>
+    <string name="OK">OK</string>
+    <string name="certificate_error_message">En säker anslutning kan inte upprättas.</string>
+    <string name="certificate_error_title">Certifikatet kunde inte läggas till.</string>
+    <string name="dlg_err_connection_failed">Anslutning misslyckades</string>
+    <string name="dlg_err_failed_unknown_host">Okänd adress: %s</string>
+    <string name="dlg_err_failed_timeout">Servern svarar inte (Timeout).</string>
+    <string name="dlg_err_failed_connect_failure">Okänt anslutningsfel</string>
+    <string name="dlg_err_failed_not_found">Ogiltligt schema URL</string>
+    <string name="dlg_err_failed_parse_failure">Kunde inte tolka svaret</string>
+    <string name="dlg_err_failed_ssl_failure">SSL misslyckades</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HHTP är inte tillåtet. Använd istället HTTPS.</string>
+    <string name="uptodate">Schemat är uppdaterat.</string>
+    <string name="schedule_changes_dialog_browse">Utforska</string>
+    <string name="schedule_changes_dialog_later">Senare</string>
+    <string name="progress_loading_data">Hämta schema &#8230;</string>
+    <string name="progress_processing_data">Hanterar schema &#8230;</string>
+    <string name="load_data">Ladda</string>
+    <string name="choose_day">Välj dag</string>
+    <string name="day">Dag</string>
+    <string name="fahrplan">Schema</string>
+    <string name="appVersion">App Version %s</string>
+    <string name="schedule_updated">Schemat har blivit uppdaterat.</string>
+    <string name="settings">Inställningar</string>
+    <string name="session_details_section_title_links">Länkar</string>
+    <string name="choose_alarm_time">Välj alarm tid</string>
+    <string name="reminders">Alarm</string>
+    <string name="usage">Tips: Håll inne på en lektion för att sätta ett alarm.</string>
+    <string name="libraries">"
+		Denna appen använder följande bibliotek:
+		Kotlin, Google Support Library, BetterLinkMovementMethod, Email Intent Builder,
+		Engelsystem, Espresso, OkHttp, SnackEngage, TraceDroid"
+	</string>
```

Comment:
Please remove the wrapping double quotes - it causes a misindentation in the app about screen.